packages/gatsby-theme/src/components/Skip/SkipContent.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+
+const SkipContent = ({ children, id, ...props }) => {
+  return (
+    <Box id={id} {...props}>
+      {children}
+    </Box>
```

Comment:
I think that we can leave this `id` as part of the components (`SkipContent` and `SkipLink`) and remove it from the props unless that there is another reason to have it as a prop.
```suggestion
    <Box id="main-content" {...props}>
      {children}
    </Box>
```

Reply:
@luisduenas The id can be changed because we can use the component to skip the content and other links, for example, Skip to navigation. 👀

Reply:
In that case we can make it required and pass it on each implementation, I don't think it's a good idea to have it as a default prop.

Reply:
You right, I'm going to change it! 🙏